portfolio/src/main/webapp/script.js

```diff
@@ -27,17 +27,21 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-// /** Redirects to the "Home" page. */
-// function goToHomePage() {
-//     location.href = 'index.html';
-// }
-
-// /** Redirects to the "Contact" page. */
-// function goToContactPage() {
-//     location.href = 'contact.html';
-// }
-
-// /** Redirects to the "Contact" page. */
-// function goToContactPage() {
-//     location.href = 'contact.html';
-// }
+// Get all comments and print them on index.html
+async function updateCommentSection() {
+    const response = await fetch('/data');
+    var commentList = await response.text();
+    
+    // Split the comment list in comments
+    commentList = commentList.slice(1, -2);
```

Comment:
This looks a bit magical. What does slice(1, -2) do?

Reply:
The string that I receive from the "fetch" function has the structure:
[elem1, elem2, ..., elem]
slice(1, -2) removes the first character ("[") and the last 2 characters ("]\0")
I am doing this so that I could use the "split" function to get every element of the string

Reply:
The string you receive is json, and there is existing code to handle it - response.json() will return JS object you can work with.